src/sentry/api/base.py

```diff
@@ -192,6 +193,8 @@ def _create_api_access_log(self):
                 path=self.request.path,
                 caller_ip=self.request.META.get("REMOTE_ADDR"),
                 user_agent=self.request.META.get("HTTP_USER_AGENT"),
+                db_queries=len(connection.queries),
```

Comment:
will `connection.queries` be populated in production? https://docs.djangoproject.com/en/2.2/faq/models/